src/core/seat/pointer.cpp

```diff
@@ -520,8 +526,19 @@ void wf::pointer_t::handle_pointer_motion_absolute(
 void wf::pointer_t::handle_pointer_axis(wlr_event_pointer_axis *ev,
     input_event_processing_mode_t mode)
 {
-    bool handled_in_binding = input->get_active_bindings().handle_axis(
-        seat->get_modifiers(), ev);
+    uint32_t modifiers = seat->get_modifiers();
+
+    bool handled_in_binding = false;
+    auto view = seat->keyboard_focus.get();
+    if (!input->active_grab && modifiers &&
```

Comment:
Because a modifier + mouse action should be handled by the application too, or this kind of action may be handled by wayfire, e.g. move or resize the virtual machine's window itself rather than the virtual OS inside it.